Gemfile

```diff
@@ -11,7 +11,7 @@ gem 'jbuilder', '~> 2.0'
 gem 'sdoc', '~> 0.4.0', group: :doc
 
 gem 'redis'
-gem 'actionkit_connector', github: 'SumOfUs/actionkit_connector', branch: 'master'
+gem 'actionkit_connector', git: 'https://github.com/SumOfUs/actionkit_connector.git', branch: 'master'
```

Comment:
I keep thinking we should publish this gem and version it (food for thought).

Reply:
Yeah, that would make sense.